.github/workflows/labeler.yml

```diff
@@ -1,11 +1,12 @@
 name: "Pull Request Labeler"
 on:
-- pull_request_target
+  pull_request_target:
+    types: [opened]
 
 jobs:
   triage:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/labeler@v4.0.3
+    - uses: actions/labeler@v3
```

Comment:
why not use the latest?

Reply:
we have been using this version all across the service repos under uc-cdis. So sticking with one version